docs/features/writing-different-telemetry-types.md

```diff
@@ -24,6 +24,64 @@ This feature requires to install our NuGet package
 PM > Install-Package Arcus.Observability.Telemetry.Core
 ```
 
+## Dependencies
+
+Dependencies allow you to track how your external dependencies are doing to give you insights on performance and error rate.
+
+We provide support for the following dependencies:
+
+- [HTTP](#measuring-http-dependencies)
+- [SQL](#measuring-sql-dependencies)
+
+### Measuring HTTP dependencies
+
+Here is how you can report a HTTP dependency:
+
+```csharp
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Tenant", "Contoso"},
+};
+
+// Create request
+var request = new HttpRequestMessage(HttpMethod.Post, "http://requestbin.net/r/ujxglouj")
+{
+    Content = new StringContent("{\"message\":\"Hello World!\"")
+};
+
+// Start measuring
+var startTime = DateTimeOffset.UtcNow;
+durationMeasurement.Start();
+
+// Send request to dependant service
+var response = await httpClient.SendAsync(request);
+
+_logger.LogHttpDependency(request, response.StatusCode, startTime, durationMeasurement.Elapsed, telemetryContext);
+// Output: "HTTP Dependency requestbin.net for POST /r/ujxglouj completed with 200 in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: [Tenant, Contoso])"
+```
+
+### Measuring SQL dependencies
+
+Here is how you can report a SQL dependency:
+
+```csharp
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Catalog", "Products"},
+    { "Tenant", "Contoso"},
+};
+
+// Start measuring
+var startTime = DateTimeOffset.UtcNow;
```

Comment:
With scoped log entries or so, we could maybe have something that does this for us.

Reply:
See #46 ;)